src/sentry/search/snuba/executors.py

```diff
@@ -305,6 +306,7 @@ class PostgresSnubaQueryExecutor(AbstractQueryExecutor):
         "first_seen",
```

Comment:
What we could do here is centralize the common fields between `postgres_only_fields` and `skip_snuba_fields`, and just add any specific fields here. That was we don't have to remember to update this list in two places.